devops/docker/Groovy/Dockerfile

```diff
@@ -0,0 +1,68 @@
+# Refer: https://github.com/groovy/docker-groovy/blob/master/jdk11/Dockerfile
+FROM adoptopenjdk:11-jdk-hotspot
+
+CMD ["groovysh"]
+
+# Set GROOVY_HOME path to place of groovy installation
+ENV GROOVY_HOME /opt/groovy
+
+RUN set -o errexit -o nounset \
+    && echo "Adding groovy user and group" \
+    && groupadd --system --gid 1000 groovy \
+    && useradd --system --gid groovy --uid 1000 --shell /bin/bash --create-home groovy \
+    && mkdir --parents /home/groovy/.groovy/grapes \
+    && chown --recursive groovy:groovy /home/groovy \
+    && chmod --recursive 1777 /home/groovy \
+    \
+    && echo "Symlinking root .groovy to groovy .groovy" \
+    && ln --symbolic /home/groovy/.groovy /root/.groovy
+
+VOLUME /home/groovy/.groovy/grapes
+
+WORKDIR /home/groovy
+
+# Installing packages and dependencies to be used in container
+RUN apt-get update \
+    && echo "Installing build dependencies" \
+    && apt-get install --yes --no-install-recommends \
+        dirmngr \
+        fontconfig \
+        gnupg \
+        unzip \
+        wget \
+        # Following 2 commands are different from actual dockerfile from https://github.com/groovy/docker-groovy/blob/master/jdk11/Dockerfile
+        # Installing git and maven to container for our requirement
+        git \
+        maven \
+    && rm --recursive --force /var/lib/apt/lists/*
+
+# Current latest version is 3.0.8. Only update this version and publish containter in future to use latest groovy
+ENV GROOVY_VERSION 3.0.8
+
+# Downloading and installing groovy
+RUN set -o errexit -o nounset \
+    && echo "Downloading Groovy" \
+    # Downloading official groovy from apache
+    && wget -q --show-progress --output-document=groovy.zip "https://archive.apache.org/dist/groovy/${GROOVY_VERSION}/distribution/apache-groovy-binary-${GROOVY_VERSION}.zip" \
+    \
+    && echo "Installing Groovy" \
+    && unzip groovy.zip \
+    && rm groovy.zip \
+    && mv "groovy-${GROOVY_VERSION}" "${GROOVY_HOME}/" \
+    # Creating Symlinks for groovy to access source directory indirectly
+    && ln --symbolic "${GROOVY_HOME}/bin/grape" /usr/bin/grape \
+    && ln --symbolic "${GROOVY_HOME}/bin/groovy" /usr/bin/groovy \
+    && ln --symbolic "${GROOVY_HOME}/bin/groovyc" /usr/bin/groovyc \
+    && ln --symbolic "${GROOVY_HOME}/bin/groovyConsole" /usr/bin/groovyConsole \
+    && ln --symbolic "${GROOVY_HOME}/bin/groovydoc" /usr/bin/groovydoc \
+    && ln --symbolic "${GROOVY_HOME}/bin/groovysh" /usr/bin/groovysh \
+    && ln --symbolic "${GROOVY_HOME}/bin/java2groovy" /usr/bin/java2groovy \
+    \
+    && echo "Editing startGroovy to include java.xml.bind module" \
+    && sed --in-place 's|startGroovy ( ) {|startGroovy ( ) {\n    JAVA_OPTS="$JAVA_OPTS --add-modules=ALL-SYSTEM"|' "${GROOVY_HOME}/bin/startGroovy"
+
+USER groovy
+
+RUN set -o errexit -o nounset \
+    && echo "Testing Groovy installation" \
```

Comment:
this is a copy-paste of official groovy image dockerfile.

It is way easier to make it like
```
FROM <groovy image name>
RUN <installation of git and whatever we need>
```
it should work as well

Reply:
Done